internal/pkg/deploy/deploy.go

```diff
@@ -87,6 +95,82 @@ func NewStore(sessProvider SessionProvider, store ConfigStoreClient) (*Store, er
 	return s, nil
 }
 
+// Pipeline is a deployed pipeline.
```

Comment:
Was there a reason why you preferred adding this functionality to the `deploy` pkg vs. `describe`?

I find both of these packages today confusing with what responsibility they have